tamr_toolbox/utils/operation.py

```diff
@@ -128,3 +128,31 @@ def wait(
             return operation
         operation = operation.poll()
     raise TimeoutError(f"Waiting for operation took longer than {timeout_seconds} seconds.")
+
+
+def monitor(
+    operation: Operation, *, poll_interval_seconds: int = 1, timeout_seconds: Optional[int] = None,
```

Comment:
What happens for default timeout_seconds = None? Probably should use a default numeric value instead, e.g. 60 seconds

Reply:
The operation would hang "forever". I updated the default to be 300 seconds